roles/metallb/tasks/main.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Copy MetalLB manifest
+  ansible.builtin.template:
+    src: "metallb-crds.yaml.j2"
+    dest: "{{ k3s_server_location }}/server/manifests/metallb-crds.yaml"
+    owner: root
+    group: root
+    mode: '0644'
+
+- name: Create temp manifests directory for metallb configuration
+  ansible.builtin.file:
+    path: "/tmp/k3s/metallb"
+    state: directory
+    owner: "{{ ansible_user_id }}"
+    mode: '0755'
+
+- name: Copy MetalLB CRs manifest
```

Comment:
I would recommend that it's worth converting to native ansible k8s module, it has support to wait for resource creation so you can avoid the "sleep 10" below.

Reply:
I think that may be a future addition if we continue to use/grow this, need to minimize scope creep for now.